docs/standard/io/pipelines.md

```diff
@@ -338,8 +339,18 @@ The previous method of writing uses the buffers provided by the `PipeWriter`. Al
 
 The <xref:System.IO.Pipelines.IDuplexPipe> is a contract for types that support both reading and writing. For example, a network connection would be represented by an `IDuplexPipe`.
 
- Unlike `Pipe` which contains a `PipeReader` and a `PipeWriter`, `IDuplexPipe` represents a single side of a full duplex connection. That means what is written to the `PipeWriter` will not be read from the `PipeReader`.
+ Unlike `Pipe`, which contains a `PipeReader` and a `PipeWriter`, `IDuplexPipe` represents a single side of a full duplex connection. That means what is written to the `PipeWriter` will not be read from the `PipeReader`.
 
 ## Streams
 
-When reading or writing stream data, you typically read data using a de-serializer and write data using a serializer. Most of these read and write stream APIs have a `Stream` parameter. To make it easier to integrate with these existing APIs, `PipeReader` and `PipeWriter` expose an <xref:System.IO.Pipelines.PipeReader.AsStream%2A>.  <xref:System.IO.Pipelines.PipeWriter.AsStream%2A> returns a `Stream` implementation around the `PipeReader` or `PipeWriter`.
+When reading or writing stream data, you typically read data using a de-serializer and write data using a serializer. Most of these read and write stream APIs have a `Stream` parameter. To make it easier to integrate with these existing APIs, `PipeReader` and `PipeWriter` expose an <xref:System.IO.Pipelines.PipeReader.AsStream%2A> method. <xref:System.IO.Pipelines.PipeWriter.AsStream%2A> returns a `Stream` implementation around the `PipeReader` or `PipeWriter`.
+
+### Stream example
+
+`PipeReader` and `PipeWriter` instances can be created using the static `Create` methods given a <xref:System.IO.Stream> object and optional corresponding creation options.
```

Comment:
- We should also talk about the overloads that take options and what the options are. (StreamPipeReaderOptions and StreamPipeWriteOptions).
- We should discuss ownership of object lifetime. How completing a PipeReader or PipeWriter returned from Create affects dispose and dispose async.

Reply:
I'll add the details about the options, I have everything I need for that.

Curious however about the ownership of object lifetime, what affects are you referring to specifically? The streams are obviously disposable/async disposable.

Reply:
The leaveOpen flag that will by default dispose the stream when you complete the `PipeReader` or `PipeWriter`.